src/uniprotkb/components/protein-data-views/__tests__/CSVView.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import { screen, render } from '@testing-library/react';
+import CSVView from '../CSVView';
+
+describe('CSVView', () => {
+  it('should return nothing if no data provided', () => {
+    const { container } = render(<CSVView data={undefined} />);
+    expect(container.firstChild).toBeNull();
```

Comment:
```suggestion
    expect(container).toBeEmptyDOMElement();
```
Is also available for this